contrib/upldrel/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/libpod/imgts:latest
+
+RUN yum -y update && \
+    yum -y install unzip && \
+    yum clean all
```

Comment:
You should add an rpm -V unzip after the yum install.  Yum doesn't error out in all cases when an install fails.

Reply:
oh right, it's a good idea, I'll add that.